src/scripts/fetch_user_handles.sh

```diff
@@ -29,7 +29,7 @@ function run_main() {
         GITHUB_GET_PR=$(curl -s "${GITHUB_API}/pulls/${GITHUB_PR_NUMBER}" \
         -H "Authorization: Bearer ${GITHUB_TOKEN}")
         # get the author of that pr
-        PR_AUTHOR=$(echo "$GITHUB_GET_PR" | jq '.user.login')
+        PR_AUTHOR=$(echo "$GITHUB_GET_PR" | jq '.user.login' | tr -d '"')
```

Comment:
You can remove the quotes by running jq "raw" with `jq -r ...` -- no need to use `tr`.